src/include/optimizer/filter_push_down_optimizer.h

```diff
@@ -10,14 +10,19 @@ namespace optimizer {
 
 struct PredicateSet {
     binder::expression_vector equalityPredicates;
+    binder::expression_vector unEqualityPredicates;
     binder::expression_vector nonEqualityPredicates;
```

Comment:
Since we have equality & unEquality, nonEquality is no longer a good naming. Let's rename this to otherPredicates